ci/ctest_to_gitlab.sh

```diff
@@ -93,7 +93,7 @@ JOB_TEMPLATE="
     USE_MPI: 'YES'
     DISABLE_AFTER_SCRIPT: 'YES'
     DLAF_HDF5_TEST_OUTPUT_PATH: /dev/shm
-  script: mpi-ctest -L {{CATEGORY_LABEL}} -L {{RANK_LABEL}}
+  script: stdbuf -oL -eL mpi-ctest -L {{CATEGORY_LABEL}} -L {{RANK_LABEL}}
```

Comment:
out of curiosity: why do you prefer line-buffered rather than no buffer at all (with mode 0)?

Reply:
We use grep inside the script to remove the lines we don't need, the output is anyway line buffered.